front/config.php

```diff
@@ -37,8 +37,7 @@
 //header("Location:../../central.php");
 
 // Entry menu case
-define('GLPI_ROOT', '../..');
-include (GLPI_ROOT . "/inc/includes.php");
+include ("../../../inc/includes.php");
```

Comment:
This file must move in front/ folder or GLPI compute a broken relative path to find a twig template